client/src/components/base/Preview.tsx

```diff
@@ -56,7 +56,7 @@ export default function Preview({
         navigate(slugs.grants);
         localResetStatus();
         dispatch(
-          addAlert("success", "Your project has been saved successfully!")
+          addAlert("success", "Your project has been saved successfully!", "")
```

Comment:
I would actually pass `undefined` instead of empty string so it feels better when we check if there's a body or not